frontend/src/app/pages/home/home.component.css

```
@@ -1,15 +1,29 @@
 .container {
-  height: 100svh;
-  height: 100vh;
+  min-height: 100vh;
```

```suggestion
  min-height: 100svh;
  min-height: 100vh;
```
Bitte mach das so wie oben.